build/build/src/ide/web/download.rs

```diff
@@ -0,0 +1,105 @@
+//! Support for downloading files.
```

Comment:
I believe this effectively duplicates `build/ci_utils/src/cache/download.rs`, including the use-case of using octocrab to download the file?
I think these should be unified and be part of `ide_ci::cache::download`.

The `Downloader` is neat. I don't like the assumption of contents being constant by URL (as my use-case is a counterexample) — but perhaps the `Key` type could be provided by the `Downloader` as well?

Reply:
Oh, I missed that module. Rewrote it to use the existing API.